Makefile

```
@@ -135,7 +135,7 @@ PROPAGATE_VARS := \
 
 GO_ENV := GOOS=$(GOOS) GOARCH=$(GOARCH) GOARM=$(GOARM) CGO_ENABLED=$(CGO_ENABLED)
 
-VERSION      ?= $(shell ./tools/image-tag)
+VERSION      := $(shell bash ./tools/image-tag)
```

`?=` is significant here; we don't want to override VERSION if it's set via the environment; this is used in our CI scripts. 

```suggestion
VERSION      ?= $(shell bash ./tools/image-tag)
```